src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModal.tsx

```diff
@@ -0,0 +1,36 @@
+import { Join, ModalDialog, Separator, Spacer } from "@artsy/palette"
+import { FC } from "react"
+import { SuggestedArtworksModalHeader } from "Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModalHeader"
+import { SuggestedArtworksModalGridQueryRenderer } from "Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModalGrid"
+import { useTranslation } from "react-i18next"
+import { useSavedSearchAlertContext } from "Components/SavedSearchAlert/SavedSearchAlertContext"
+
+interface SuggestedArtworksModalProps {
+  onClose: () => void
+}
+
+export const SuggestedArtworksModal: FC<SuggestedArtworksModalProps> = ({
+  onClose,
+}) => {
+  const { t } = useTranslation()
+  const { criteria } = useSavedSearchAlertContext()
+
+  return (
+    <ModalDialog
+      width={["100%", 700]}
```

Comment:
We can remove the width since the default already works as we want it to, it should be the same as the Create Alert modal.

Reply:
The reason why I use this `width` is that we have a modal (confirmation step after alert has been created), which looks almost identical to this one. I think we should keep styling between them aligned, wdyt?

Also without width modal looks a bit narrow:

![Screenshot 2023-08-11 at 10 45 57](https://github.com/artsy/force/assets/3934579/5265bfd9-726e-4540-842c-771562b449a8)

Reply:
Yeah I see, I guess that's a general question then - if we want to have custom heights (unless it's mandatory for some functionality) since it doesn't make a huge difference. We can move forward and address async if necessary.